cli/config/jest.styled-jsx-css.mock.js

```diff
@@ -0,0 +1,5 @@
+function css() {
+    return ''
+}
+
+module.exports = css
```

Comment:
I reckon we'll also need to moch `resolve`

Reply:
The article doesn't mention this specifically, so perhaps it's not required... But we do tend to use `resolve` and I think it probably needs the same treatment as `css`.

Reply:
Thanks @HendrikThePendric, good catch - I added both `resolve` and `global` functions and also mocked the `macro` entrypoint in 2fac1ce